src/components/sidebar/sidebar.components.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./sidebar.components.scss";
+import chatIcon from "./messenger.png";
+import userIcon from "./user.png";
+import moreIcon from "./more.png";
+
+import { Link } from "react-router-dom";
+
+function SideBar() {
+  return (
+    <>
+      <div className="sidenav">
+        <Link to={`/profile`}>
+          <img src={userIcon} className="user" alt="user" />
+        </Link>
+        <Link to={`/chatroom`}>
+          <img src={chatIcon} className="chat" alt="chat" />
+        </Link>
+        <Link to={`/other`}>
+          <img src={moreIcon} className="more" alt="more" />
+        </Link>
+      </div>
+    </>
```

Comment:
```suggestion
      <div className="sidenav">
        <Link to={`/profile`}>
          <img src={userIcon} className="user" alt="user" />
        </Link>
        <Link to={`/chatroom`}>
          <img src={chatIcon} className="chat" alt="chat" />
        </Link>
        <Link to={`/other`}>
          <img src={moreIcon} className="more" alt="more" />
        </Link>
      </div>
```
여기에서도 `fragment`는 사용하지 않아도 될 것 같아요